src/createListComponent.js

```diff
@@ -457,11 +458,15 @@ export default function createListComponent({
           return null;
         }
 
+        const isRtl = this.props.style && this.props.style.direction === 'rtl';
+
         return {
           isScrolling: true,
           scrollDirection:
             prevState.scrollOffset < scrollLeft ? 'forward' : 'backward',
-          scrollOffset: scrollLeft,
+          scrollOffset: isRtl
+            ? scrollWidth - clientWidth - scrollLeft
+            : scrollLeft,
```

Comment:
I don't know if this is right. Why should we invert the element's `scrollLeft`? That seems...wrong. Am I thinking about this wrong?

Reply:
Hm... yeah, this seems incorrect (and unnecessary) to me.

Reply:
Sorry. I think I was mistaken.